util/inficon/inficonvgc502.py

```
@@ -0,0 +1,54 @@
+import asyncio
+import sys
+
+class InficonVGC502:
+    def __init__(self, address, port, timeout=1):
+        self.address = address
+        self.port = int(port)
+        self.timeout = timeout
+        self.reader = None
+        self.writer = None
+
+    async def __aenter__(self):
+        try:
+            self.reader, self.writer = await asyncio.open_connection(self.address, self.port)
+        except ConnectionRefusedError as err:
+            print(f"Connection refused: {err}")
+            sys.exit(1)
```

Could you raise a custom exception here instead of terminating the application with sys.exit()?